NAMESPACE

```diff
@@ -32,7 +32,9 @@ export(edit_r_profile)
 export(edit_rstudio_prefs)
 export(edit_rstudio_snippets)
 export(edit_template)
+export(gh_lock_branch)
```

Comment:
What do we think of `github_lock_branch()`? Just want us to at least consider. I don't have a strong opinion.